Makefile

```diff
@@ -26,6 +26,10 @@ DOCKER := docker
 SKOPEO := skopeo
 PODMAN := podman
 
+# The default e2e test type is set to "ci-tests" if not provided
+TEST_TYPE ?= ci-tests
+TEST_OPTIONS ?= ""
+
```

Comment:
What do you think of making it simpler, by introducing just one variable, like `E2E_OPTIONS ?= ci-tests` ?
With that, the `test-e2e` target below, could be simplified into something like:

```
$(MAKE) -C tests/ $(E2E_OPTIONS)
```

And the execution could be done as:

```
make test-e2e
make test-e2e E2E_OPTIONS="e2e-tests"
make test-e2e E2E_OPTIONS="test TEST=hello-world"
```